farm/modeling/prediction_head.py

```diff
@@ -1103,14 +1104,14 @@ def formatted_preds(self, logits, baskets, rest_api_schema=False):
         preds_d = self.aggregate_preds(preds_p, passage_start_t, ids, seq_2_start_t)
         assert len(preds_d) == len(baskets)
 
-        # Separate top_preds list from the no_ans_gap float
-        top_preds, no_ans_gaps = zip(*preds_d)
+        # Separate top_preds list from the change_boost float. Add change_boost to current boost for switching
+        top_preds, change_boosts = zip(*preds_d)
```

Comment:
I was first confused by the naming of "change_boosts" here. I would vote for keeping "no_ans_gap" or something like "no_ans_diff"